validphys2/src/validphys/pdfplots.py

```diff
@@ -273,6 +273,104 @@ def plot_pdf_uncertainties(
     PDF's central value is plotted. Otherwise it is the absolute values."""
     yield from UncertaintyPDFPlotter(pdfs, xplotting_grids, xscale, normalize_to, ymin, ymax)
 
+class PullPDFPlotter(metaclass=abc.ABCMeta):
+    """Auxiliary class which groups multiple pulls in one plot."""
+
+    def __init__(self, pdfs_list, pull_grids_list, xscale, normalize_to, ymin, ymax):
+        self.pdfs_list = pdfs_list
+        self.pull_grids_list = pull_grids_list
+        self._xscale = xscale
+        self.normalize_to = normalize_to
+        self.ymin = ymin
+        self.ymax = ymax
+        self.firstgrid = pull_grids_list[0][0]
```

Comment:
Please add a docstring for these inputs

Reply:
[4a24968](https://github.com/NNPDF/nnpdf/pull/1910/commits/4a249681043708b4e0ece94924c8e502be2d6b58)